Entities/Ingame/Club.cs

```diff
@@ -48,13 +48,13 @@ public Club (int memoryAddress, ArraySegment<byte> originalBytes, IVersion versi
 
 		public string Name { 
 			get {
-				return PropertyInvoker.GetString (ClubOffsets.Name, 0, OriginalBytes, MemoryAddress, DatabaseMode);
+				return PropertyInvoker.GetString (ClubOffsets.Name, -1, OriginalBytes, MemoryAddress, DatabaseMode);
 			}
 		}
 
 		public string ShortName {
 			get {
-				return PropertyInvoker.GetString (ClubOffsets.ShortName, 0, OriginalBytes, MemoryAddress, DatabaseMode);
+				return PropertyInvoker.GetString (ClubOffsets.ShortName, -1, OriginalBytes, MemoryAddress, DatabaseMode);
 			}
 		}
 
```

Comment:
It shouldn't, as I changed ProcessManager.GetString to read the extra pointer "if addBufferIndex >= 0" instead of "if addBufferIndex > 0". So there should be no difference (unless I'm being stupid). I did the ProcessManager.GetString change  for Person.Firstname and Person.Lastname as it seems they need to read the extra pointer but the addBufferIndex needs to be 0. 

I just realised there will be a problem with other versions with Person.Firstname and Person.Lastname since I changed the addBufferIndex for them, it was 0xC before but now it needs to be 0.. I'll fix this soon !